docs/Configuration.md

```diff
@@ -0,0 +1,44 @@
+# Configuration
+
+Zonemaster Backend is configured as a whole from `/etc/zonemaster/backend_config.ini`.
+
+Each section in `backend_config.ini` is documented below.
+
+## DB section
+
+TBD
+
+
+## GEOLOCATION section
+
+TBD
+
+
+## LOG section
+
+TBD
+
+
+## PERL section
+
+TBD
+
+
+## PROFILES section
+
+The PROFILES section declares the available [profiles].
+
+Keys are [profile names], and values are absolute file system paths to [profile JSON files].
+The profile named `default` is special and always available.
+When it isn't explicitly defined, it's implicitly mapped to the Zonemaster Engine default profile.
```

Comment:
My picture is the following:

1. The Engine default profile is always loaded.
2. If Backend defines profiles (symbolic name to a profile file), and a such a profile is selected, then that is used to override what is been defined by Engine (on top). If selected profile name or profile file is unavailable, backend will return an error.
3. Profile "default" can always be selected. If there is a profile file defined for "default", then that is loaded on on top of the Engine profile. If there is no such file defined, then that is just silently ignored.